src/media/js/addon/containers/dashboardDetail.js

```diff
@@ -45,12 +45,12 @@ export class AddonDashboardDetail extends React.Component {
 
   renderDeleted() {
     return (
-      <div>
-        <h2>This Firefox OS add-on has been deleted.</h2>
-        <ReverseLink to="addon-dashboard">
-          Return to My Firefox OS Add-ons
-        </ReverseLink>
-      </div>
+      <PageSection>
+        <p>
+          This add-on has been deleted. <ReverseLink to="addon-dashboard">
```

Comment:
I don't like nbsp, since that will prevent a line break on that space, where one might actually belong in some circumstances.